jira/client.py

```diff
@@ -674,6 +674,9 @@ def __init__(
 
         self._fields_cache_value: dict[str, str] = {}  # access via self._fields_cache
 
+        # Store fields cache reference on session for display name field injection
+        self._session.fields_cache = self._fields_cache
```

Comment:
This feels like a hack to me, but I could not find a way to access _fields_cache from resources directly, but I would love to be pointed in the 'right' way to do it.